news/2017-06-27-beatmap-spotlight-may-2017.md

```diff
@@ -0,0 +1,124 @@
+I see sets like this more and more rarely as time passes, which is a shame. All the four mappers involved in the set did a splendid job in using the unusual doubles in this song and provided (although probably nothing exceptional) a really nice and clean approach to mapping. Overall this mapset is not pretentious at all, but that's most likely the reason why it is here, to demonstrate that not necessarily heavy rhythms or intense jump sessions are important to create an entertaining mapset that hooks the player. Whether you are an expert player or just a casual one you'll have something to chew on considering it is easily DT-able (not sure how many pp it gives tho :P). I could keep praising this mapset for a while but let's say that you should definitely give it a go and witness some old-school mapping modernized for the new standards.
+
+[![](/wiki/shared/news/2017-06-27-beatmap-spotlights-may-2017/ben-se.jpg)](https://osu.ppy.sh/s/567325)
+
+This mapset truly stands out due to the unique spacing and patterning in both difficulties. The flow is smooth but expressive, perfectly supporting the mood of the song. 09kami's and Crystal's maps are on a similar difficulty level, but represent the song in different ways. Check out both of them, you will not regret it!
+
+[![](/wiki/shared/news/2017-06-27-beatmap-spotlights-may-2017/kokoronashi.jpg)](https://osu.ppy.sh/s/565432)
+
+Kokoronashi is yet another great mapset contributed by pishifat. Playing the map feels unique, since it avoids rotational flow and focuses entirely on back-and-forth or linear patterns. You should definitely check this one out if you haven't yet!
+
+[![](/wiki/shared/news/2017-06-27-beatmap-spotlights-may-2017/yuki-no-hana.jpg)](https://osu.ppy.sh/s/565432)
+
+A calm, beautiful winter song with a very good-looking skin and a very well designed mapset. While the top difficulty is quite spammy in terms of object density, the visual aspect of the map does not fall apart! Combined with great lower difficulties, this mapset deserves to be featured in the spotlights.
+
+## osu!taiko
+
+[![](/wiki/shared/news/2017-06-27-beatmap-spotlights-may-2017/classic-pursuit.jpg)](https://osu.ppy.sh/s/373254)
+
+Spooky! Just 7 months after Halloween Hanjamon's Classic Pursuit entered the ranked section, and what a great map it is! Featuring a huge variety of Slider Velocity changes, tricky off-beat patterns, consistency (which does not equal boring) and even 1/8, this map offers a lot of things to enjoy. And if the map is too easy for you: Try it with ANY mod. It's a challenge with Hidden, Hardrock and Doubletime, with the variety of Slider Velocities and complex 1/4+1/8 keeping you on your toes no matter what mod you are using. The set also has nicely done lower diffs, including Hanjamon's signature "Basic" difficulty, which introduces you into the most basic structures of osu!taiko's dons and kats (the red and blue notes!). We hope that you will enjoy this map - and replay it during Halloween!
```

Comment:
By Nwolf petition, he would like to change the comment to:

+Spooky! Just 7 months after Halloween Hanjamon's Classic Pursuit entered the ranked section, and what a great map it is! Featuring a huge variety of Slider Velocity changes, tricky off-beat patterns, consistency (which does not equal boring) and even 1/8, this map offers a lot of things to enjoy. And if the map is too easy for you: Try it with ANY mod. It's a challenge with Hidden, Hardrock and Doubletime, with the variety of Slider Velocities and complex 1/4+1/8 keeping you on your toes no matter what mod you are using. 

The set also has nicely done lower diffs, including Hanjamon's signature "Basic" difficulty, which introduces you into the most basic structures of osu!taiko's dons and kats (the red and blue notes!). We hope that you will enjoy this map - and replay it during Halloween!